jsettlers.logic/src/main/java/jsettlers/input/GuiInterface.java

```diff
@@ -397,6 +399,11 @@ public void action(IAction action) {
 			case UNLOAD_FERRIES:
 				unloadFerries();
 				break;
+				
+			case SET_MOVEABLE_RATIO:
+				SetMoveableRatioAction ratioAction = (SetMoveableRatioAction) action;
```

Comment:
This is where the actual ratio change should take place.
Look at SPEED_FASTER/SPEED_SLOWER for reference